### ISSUE TITLE
Cannot access Status message of SVProgressHUD from XCTest

### ISSUE THREAD

**Issue by binsnoel** [SVProgressHUD/SVProgressHUD#837]

**Comment by honkmaster** [comment]
IF you want to access this propery, make a pull request.

**Comment by rpendleton** [comment]
@binsnoel This is happening because the SVProgressHUD is marked as an accessible element, but the accessibilityValue is being set on the hudView itself. I don't know enough about the codebase to fix this (and I haven't found a need to verify status text in my project), but you could fix this by ensuring the accessibiltiyLabel has been set on the outermost element that has accessibility enabled.

Given that the outermost accessibility element doesn't have a value set, I'd assume Voice Over actually struggles with reading the label of the HUD, but that might be okay since notifications are posted when the status changes.
